tarantool/response.py

```diff
@@ -17,13 +18,16 @@
     IPROTO_ERROR,
     IPROTO_SYNC,
     IPROTO_SCHEMA_ID,
-    REQUEST_TYPE_ERROR
+    REQUEST_TYPE_ERROR,
+    IPROTO_SQL_INFO,
+    IPROTO_SQL_INFO_ROW_COUNT,
+    IPROTO_SQL_INFO_AUTOINCREMENT_ID
```

Comment:
It is named as SQL_INFO_AUTOINCREMENT_IDS in [execute.h](https://github.com/tarantool/tarantool/blob/a1021237739b2c24be1252139fd66eae32c24884/src/box/execute.h#L45) and it is a list of keys. So I would keep trailing 'S'.

Reply:
Fixed